serverless/functions/viewer-connected-to-player.js

```diff
@@ -0,0 +1,93 @@
+/* global Twilio Runtime */
+'use strict';
+
+module.exports.handler = async (context, event, callback) => {
+  const { user_identity, stream_name } = event;
+  const common = require(Runtime.getAssets()['/common.js'].path);
+  const { getStreamMapItem } = common(context, event, callback);
+
+  if (!user_identity) {
+    response.setStatusCode(400);
+    response.setBody({
+      error: {
+        message: 'missing user_identity',
+        explanation: 'The user_identity parameter is missing.',
+      },
+    });
+    return callback(null, response);
+  }
+
+  if (!stream_name) {
+    response.setStatusCode(400);
+    response.setBody({
+      error: {
+        message: 'missing stream_name',
+        explanation: 'The stream_name parameter is missing.',
+      },
+    });
+    return callback(null, response);
+  }
+
+  let response = new Twilio.Response();
+  response.appendHeader('Content-Type', 'application/json');
+
+  const client = context.getTwilioClient();
+
+  let room, streamSyncClient;
+
+  try {
+    // See if a room already exists
+    room = await client.video.rooms(stream_name).fetch();
+  } catch (e) {
+    console.error(e);
+    response.setStatusCode(500);
```

Comment:
Nit, but thoughts on returning 404 here?

Reply:
That is logical to me but currently the backend only uses 400 and 500 error codes. Maybe it is useful to reserve 404 specifically for invalid URL. I know I run into 404 sometimes when testing and then immediately realize oh I need to update the backend URL. I could also see this being a 400 instead of 500. It probably isn't a backend error. I can go either way just some things to think about.